packages/vm/package.json

```diff
@@ -43,13 +41,13 @@
     "VM"
   ],
   "dependencies": {
-    "async-eventemitter": "^0.2.4",
-    "core-js-pure": "^3.0.1",
     "@ethereumjs/account": "^3.0.0",
     "@ethereumjs/block": "^3.0.0",
     "@ethereumjs/blockchain": "^4.0.3",
     "@ethereumjs/common": "^1.5.1",
     "@ethereumjs/tx": "^2.1.2",
+    "async-eventemitter": "^0.2.4",
+    "core-js-pure": "^3.0.1",
```

Comment:
What are these two new dependencies?

Reply:
@holgerd77 they [were already there](https://github.com/ethereumjs/ethereumjs-vm/pull/886/files/7108dafe36bc9d4d00ad14abd2006b487a0cb2af#diff-76c14e78cd18d988b3bb6794290539fbL46). I have just moved them to the correct order (@, A-Z)